### PR TITLE
NodeJS SDK: convert response type from buffer to JSON object

### PR DESCRIPTION
**Pull request by wepayer**

### CODE REVIEW

**Comment by lukeheath** [comment]
Would be great to have this merged in!

File: lib/wepay.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ exports.WEPAY = function(settings)
                 response.on('data', function(chunk) {
                     // get the last argument
                     if (callbackFunction && typeof callbackFunction === 'function') {
-                        callbackFunction(chunk);
+                        callbackFunction(JSON.parse(chunk.toString('utf8')));
```
Comment (mattuuh7) [inline lib/wepay.js]: Hi @lukeheath 
Can you please resolve the conflict then make the below changes?
We now have code to catch all the buffered response payload. All you need to do is to stringify that body and use that to call the callback, so something like this:
```
callbackFunction(JSON.parse(body.toString('utf8')));
```
Please fix that and I will merge it.
Thanks,

**Comment by lukeheath** [comment]
This PR is from @wepayer

**Comment by y2chaits** [comment]
These changes are being redone in #12 .
Closing.

**Pull request closed without merging**